core/app/models/spree/product/scopes.rb

```diff
@@ -173,6 +173,7 @@ def self.property_conditions(property)
 
     # Can't use add_search_scope for this as it needs a default argument
     def self.available(available_on = nil, _currency = nil)
+      Spree::Deprecation.warn("The second currency argument on Product.available has no effect, and is deprecated", caller) if _currency
```

Comment:
the leading underscore was what gave me a clue it _wasn't_ being used when I saw it in source (see it had been auto-added in the past by a rubocop automated fix thing), so I felt bad removing it just because I've added a deprecation warning on what's still a no-op argument! but I will since you asked, sure.